framework/common/include/mapf/common/socket.h

```diff
@@ -74,24 +74,50 @@ class SubSocket : public Socket {
         return Subscribe(topic);
     }
     int Subscribe(const std::initializer_list<std::string> &topics);
+    /**
+     * MMZ
+     * Parameters that are not modified inside the method should be passed as
+     * const reference
+     */
     template <class T> int Subscribe(std::initializer_list<std::string> topics)
     {
         mapf_assert(topics.size() > 0);
         for (const auto &topic : topics)
+          /**
+           * MMZ
+           * Method return value is ignored
+           */
             Subscribe<T>(topic);
+        /**
+         * MMZ
+         * Method should return a value
+         */
```

Comment:
This function is actually unused. I'm surprised though that gcc doesn't warn about it (and -Werror is turned on...)

Please create a hotfix PR for this.